src/net/dmytro/homework4/Animal.java

```diff
@@ -0,0 +1,13 @@
+package net.dmytro.homework4;
+
+public abstract class Animal {
```

Comment:
It is good idea to use protected fields here, but you don’t use them. 

And think about rename them to maxRun.. and maxSwim..